src/plugins/data/server/search/strategies/ese_search/ese_search_strategy.ts

```diff
@@ -170,7 +171,7 @@ export const enhancedEsSearchStrategyProvider = (
      */
     search: (request, options: IAsyncSearchOptions, deps) => {
       logger.debug(`search ${JSON.stringify(request.params) || request.id}`);
-      if (request.indexType && request.indexType !== 'rollup') {
```

Comment:
Thats assigned on `src/plugins/data/common/search/search_source/search_source.ts:778` - the searchsource code often refers to indices when really it means data views. @lukasolson might be able to add some context

Reply:
I don't think we actually need this check at all. We can just call the rollup search if `indexType` is `'rollup'` and otherwise do the normal search. In any case, this extra check shouldn't do anything anyway since we *should* never be in this search strategy with an ESQL data view anyway.

Reply:
@lukasolson I'd like to verify that I've changed the code to your liking.